templates/web/oxfordshire/around/tabbed_lists.html

```diff
@@ -0,0 +1,24 @@
+<p class="report-list-filters">
+    <label>
+        Show
+        <select>
+            <option>all reports</option>
+            <option>unfixed reports</option>
+            <option>fixed reports</option>
+        </select>
+    </label>
+    <label>
+        about
+        <select>
+            <option>anything</option>
+            <option>bridges</option>
+            <option>carriageway defect</option>
+            <option>debris/spillage</option>
+            <option>…</option>
+        </select>
```

Comment:
Is a server-side way of filtering this page going to be included here? If not, at the least these drop downs should only appear if their functionality is available. But really preferably the first option… :)